test/Sentry.Tests/SentryClientTests.cs

```diff
@@ -365,6 +366,48 @@ public void CaptureUserFeedback_DisposedClient_ThrowsObjectDisposedException()
             _ = Assert.Throws<ObjectDisposedException>(() => sut.CaptureUserFeedback(null));
         }
 
+        [Fact]
+        public void CaptureTransaction_ValidTransaction_Sent()
+        {
+            // Arrange
+            var sut = _fixture.GetSut();
+
+            // Act
+            sut.CaptureTransaction(new Transaction(DisabledHub.Instance, null)
+            {
+                Name = "test name",
+                Operation = "test operation"
+            });
+
+            // Assert
+            _ = sut.Worker.Received(1).EnqueueEnvelope(Arg.Any<Envelope>());
+        }
+
+        [Fact]
+        public void CaptureTransaction_InvalidTransaction_Ignored()
```

Comment:
Why is it ignored?

Maybe: `CaptureTransaction_NoName_Ignored`

`CaptureTransaction_NoOperation_Ignored`

and one case per each?